content/en/admin/install.md

```diff
@@ -71,10 +71,11 @@ git clone https://github.com/rbenv/ruby-build.git ~/.rbenv/plugins/ruby-build
 ```
 
 Once this is done, we can install the correct Ruby version:
+(check the latest version of Ruby on (https://www.ruby-lang.org/en/downloads/) and read documentation on rbenv
```

Comment:
Actually you should just follow this guide instead of needing to choose a Ruby version on your own. The guide was written so that you *don't* need to go and read a bunch of different documentations like the rbenv one.

Reply:
Agree. I was under impression that it’s outdated versions of old packages.

But this guide can provide more details on as it took lot of efforts for me to understand how to link domain with server e.g. 

Reference such as how to have installation on cloud vs local installation.

If cloud then how to have A record in domain service provider.

ThE reference around having installation on Raspeberry Pi. ( e.g editing version of bycrypt 1.1.13 to 1.1.12 in Gem.bak file ) 

Actually I have now enitire instructions set  on how To install Mastodon on Raspeberry Pi on local server.

Now my site is live on my LAN but unstable to access from WAN. Exploring / learning more on port forwarding etc.


AG